portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,175 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Yusuf Abdulmueez's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div class="ScrollWrapper">
+            <header class="Header js-HasVH" data-vh="0.75">
+                <nav>
+                    <a class="x" href="#about">
```

Comment:
Can there be more meaningful names for these classes (x, v, h, e, d, ....)? For example, "x" seems to be navbar-item?